src/sentry/static/sentry/app/views/teamCreate.jsx

```diff
@@ -29,6 +29,25 @@ export default class TeamCreate extends AsyncView {
     return 'Create Team';
   }
 
+  renderField() {
+    let features = new Set(this.context.organization.features);
+    return features.has('internal-catchall') ? (
+      <TextField
+        name="slug"
+        label={t('Team Slug')}
+        placeholder={t('e.g. operations, web, desktop')}
```

Comment:
Hmm, I'm guessing we slugify on backend if you use spaces? Maybe add a hyphenated placeholder example?